docs/source/indexes/cagra.rst

```diff
@@ -98,53 +98,148 @@ Tuning Considerations
 The 3 hyper-parameters that are most often tuned are `graph_degree`, `intermediate_graph_degree`, and `itopk_size`.
 
 Memory footprint
-----------------
+================
 
-CAGRA builds a graph that ultimately ends up on the host while it needs to keep the original dataset around (can be on host or device).
+CAGRA stores the dataset (raw vectors) and nearest-neighbor graph (neighbor IDs) in memory.
```

Comment:
What memory? This rewrite of the sentence adds some good additional details but seems to have lost some important information.